src/main/java/com/github/manolo8/darkbot/utils/APIErrors.java

```diff
@@ -0,0 +1,70 @@
+package com.github.manolo8.darkbot.utils;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static com.github.manolo8.darkbot.gui.utils.Popups.showMessageSync;
+
+public class APIErrors {
+
+
+    private final String RUNTIME_URL  = "https://darkbot.eu/downloads/Runtimes4DarkBot.exe";
+    private final String RUNTIME_LINK = "Download link: <a href=\"#\">" + RUNTIME_URL;
+
+    JPanel pnl = new JPanel();
+
+    private void displayAndExit() {
+        JOptionPane options = new JOptionPane(this.pnl,
+                JOptionPane.ERROR_MESSAGE,
+                JOptionPane.DEFAULT_OPTION,
+                null, new Object[]{}, null
+        );
+        showMessageSync("Error", options);
+        System.exit(0);
+    }
+
+    public void displayUnsupportedJavaError() {
+        JLabel unSuppJava    = new JLabel("<html>" + I18n.get("api.errors.unsupported_java"));
+        JLabel clickableLink = new JLabel("<html>" + RUNTIME_LINK);
+        clickableLink.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
+        clickableLink.addMouseListener(new MouseAdapter() {
+            @Override
+            public void mouseClicked(MouseEvent e) {
+                try {
+                    Desktop.getDesktop().browse(new URI(RUNTIME_URL));
+                } catch (URISyntaxException | IOException ignored) {}
+            }
+        });
```

Comment:
This is quite ugly to add as a standalone, consider adding a util JHyperLink or similar that extends JLabel and has the logic for this